e2e/integrations/integration.steps.ts

```diff
@@ -183,6 +183,15 @@ class IntegrationSteps {
   public getStepsArray (): P<any> {
     const stepFactory = new StepFactory();
     const steps = this.world.app.getElementsByClassName('parent-step');
+    const navBar = this.world.app.getElementByClassName('nav-pf-vertical');
+
+    navBar.isDisplayed().then((visible) => {
+      if (visible) {
+        log.warn('Navigation bar is not hidden.');
+        const navBarToggle = this.world.app.getElementByClassName('navbar-toggle');
+        navBarToggle.click();
+      }
+    });
 
```

Comment:
Sorry I forgot to add it :-) However I think we should consider adding check to navigation bar navigation.